pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -447,6 +448,13 @@ public void start() throws Exception {
             }
         }
 
+        if (pulsar.getConfiguration().isTransactionCoordinatorEnabled()) {
+            this.transactionBufferDeleteMarkerMonitor = Executors
```

Comment:
can you please "shutdown" this Executor service on BrokerService close ?

I also cannot find the shutdown for these other threadpools: topicPublishRateLimiterMonitor, brokerPublishRateLimiterMonitor, deduplicationSnapshotMonitor

Reply:
good catch! I will follow your advice.